packages/block-library/src/navigation/index.js

```diff
@@ -52,6 +55,26 @@ export const settings = {
 	},
 	edit,
 	save,
+	__experimentalLabel: ( { ref } ) => {
+		if ( ! ref ) {
+			return __( 'Navigation' );
```

Comment:
The block title may have changed. Example code:

```php
function change_navigation_block_name( $args, $block_type ) {
	if ( 'core/navigation' !== $block_type ) {
		return $args;
	}
	$args['title'] = __( 'My Navigation!' );
	return $args;
}
add_filter( "register_block_type_args", "change_navigation_block_name", 10, 2 );
```


Should this possibility be considered?